### PR TITLE
Create ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58926](https://cumulocity.atlassian.net/browse/MTM-58926)
Original PR: [6497](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6497)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: When the modal for maximum number of archives is cancelled by the user, an error message is no longer displayed. [GRAFT][release/y2024] (#6497)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Error message no longer displayed when cancelling maximum number of archives modal
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58926
+version: 1018.503.95
+---
+When the modal for maximum number of archives is cancelled by the user, an error message is no longer displayed. [GRAFT][release/y2024] (#6497)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, when a user cancelled the modal dialog that warned about reaching the maximum number of archives, an error message was incorrectly displayed even though no actual error had occurred. This unwanted error message has now been removed. Users can cancel the maximum number of archives warning modal without seeing an error message.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Error message no longer displayed when cancelling maximum number of archives modal
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md:3]:
```suggestion
title: Canceling the dialog on reaching the maximum number of archives no longer displays an error message
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58926
+version: 1018.503.95
+---
+In the past, when a user cancelled the modal dialog that warned about reaching the maximum number of archives, an error message was incorrectly displayed even though no actual error had occurred. This unwanted error message has now been removed. Users can cancel the maximum number of archives warning modal without seeing an error message.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-95-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md:17]:
```suggestion
In the past, when a user reached the maximum number of archives and the corresponding dialog was shown, canceling the dialog would incorrectly display an error message. This has been fixed. Now, when a user cancels the maximum number of archives dialog, no error message is shown any longer and the application behaves as expected. 
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**